modules/monitoring-granting-user-permissions-using-the-web-console.adoc

```diff
@@ -6,7 +6,7 @@
 [id="granting-user-permissions-using-the-web-console_{context}"]
 = Granting user permissions by using the web console
 
-You can grant users permissions to monitor their own projects, by using the {product-title} web console.
+You can grant users permissions for the `openshift-monitoring` project or their own projects, by using the {product-title} web console.
```

Comment:
Is something missing here? The [current docs](https://docs.openshift.com/container-platform/4.16/observability/monitoring/enabling-monitoring-for-user-defined-projects.html#granting-user-permissions-using-the-web-console_enabling-monitoring-for-user-defined-projects) say you can grant users permissions **_to monitor_** projects. Should that also appear here?

```suggestion
You can grant users permissions to monitor the `openshift-monitoring` project or their own projects,  by using the {product-title} web console.
```

Reply:
This change is based on the engineering feedback: https://github.com/openshift/openshift-docs/pull/80127#discussion_r1734110968

Reply:
Yes, I changed it with the addition of core platform monitoring, because it is technically not only monitoring (monitoring here evokes that it is just read and write access mostly, but we can also grant configuration rights)